internal/pkg/exec/docker.go

```diff
@@ -159,3 +172,59 @@ func imageName(uri, tag string) string {
 	}
 	return fmt.Sprintf("%s:%s", uri, tag)
 }
+
+// IsEcrCredentialHelperEnabled return true if ecr-login is enabled either globally or registry level
+func (c DockerCommand) IsEcrCredentialHelperEnabled(uri string) bool {
+	// Make sure the program is able to obtain the home directory
+	splits := strings.Split(uri, "/")
+	if c.homePath != "" && len(splits) > 0 {
+		// Look into the default locations
+		pathsToTry := []string{filepath.Join(".docker", "config.json"), ".dockercfg"}
+		for _, path := range pathsToTry {
+			content, err := ioutil.ReadFile(filepath.Join(c.homePath, path))
+			if err != nil {
+				// if we can't read the file keep going
+				continue
+			}
+
+			config, err := parseCredFromDockerConfig(content)
+			if err != nil {
+				continue
+			}
+
+			if config.CredsStore == CredStoreECRLogin || config.CredHelpers[splits[0]] == CredStoreECRLogin {
```

Comment:
could you update the variable here?

Reply:
I just noticed the same const is used in unit test case as well. So I have updated it back to Capitol letter so it can be get used in both the places

Reply:
Yeah but still they are in the same package right?

Reply:
You are right, been a long day I guess :) Sorry about the back and forth.